src/main/java/util/DigestMessageGenerate.java

```diff
@@ -6,11 +6,26 @@
 import java.nio.charset.StandardCharsets;
 import java.security.MessageDigest;
 import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.util.Arrays;
 
 public class DigestMessageGenerate {
```

Comment:
maybe better create more understandable name which will convey the meaning of the main goal of this class

Reply:
Ok. I'll do it in the next HW, because i need to change it here
 is EncryptMessegeUtil normal?

Reply:
maybe better EncryptPasswordUtil?

Reply:
Ok, it's good. I will change it in next HW